thebeetoken/beenest-web
# Add rating to ListingInformation

## Description
Add ratings to the information page, when available, to provide more decisional input to guests

Note that this is branched from #322 and should be reviewed/merged thereafter

## How to Test
1. Navigate to an Agoda listing (these are the only ones with ratings)
2. Expect to see star rating under listing title
3. Regression-test non-Agoda listings (should have no ratings)

Which devices did you test on?

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
When we say there are "15 reviews" we probably want to link to those reviews. That could be tricky; I didn't see anything about linking to reviews in the Agoda docs
